backend/test/api-auth.test.js

```diff
@@ -0,0 +1,66 @@
+import request from 'supertest';
+
+jest.mock('../src/services/authService');
+import { authService } from '../src/services/authService';
+import app from '../src/app';
+
+class authenticationError extends Error {
```

Comment:
@gulyaasjaanos classnames usually start with Uppercase letters
```suggestion
class AuthenticationError extends Error {
```